### PR TITLE
chore: Migration to Angular 10

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/2754
#### Please provide a brief summary of this pull request.
- IVY is still not recommended for libraries: [source](https://angular.io/guide/ivy#maintaining-library-compatibility)
image

- Schematics had to be adjusted to newer version a little bit. tested on brand new NG10 app
image

- IE11 works well there with IVY and without IVY

- Comparison for IVY [link to docs](https://docs.google.com/document/d/1ymMAG5tHrZ5vhv_x_WdrpF19X9yjnK8QvWKROC5IZZc/edit?usp=sharing) Looks likeit's better withotut IVY

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -35,8 +35,13 @@
                         "scripts": [
                             "node_modules/marked/lib/marked.js"
                         ],
-                        "es5BrowserSupport": true,
-                        "showCircularDependencies": false
+                        "showCircularDependencies": false,
```
Comment (salarenko) [inline angular.json:38]: This property `es5BrowserSupport` is deprecated?

File: angular.json [hunk 2]
```diff
@@ -35,8 +35,13 @@
...
+                        "allowedCommonJsDependencies": [
+                          "moment",
```
Comment (salarenko) [inline angular.json:40]: Isn't it missing angular CDK?

File: angular.json (hunk 1 above)
Comment (JKMarkowski) [reply angular.json:38]: `es5BrowserSupport` flag has been removed from schema.

File: angular.json (hunk 2 above)
Comment (JKMarkowski) [reply angular.json:40]: CDK is treated as default angular library. It's not required to be added.

**Pull request merged**